metpy/plots/tests/test_skewt.py

```diff
@@ -76,11 +76,10 @@ def test_hodograph_units():
     """Test passing unit-ed quantities to Hodograph."""
     with style.context(test_style):
         fig = make_figure(figsize=(9, 9))
```

Comment:
This `fig=` line can also go, since `Hodograph` will take care of making a figure.